modules/javafx.base/src/test/java/test/util/memory/JMemoryBuddy.java

```diff
@@ -91,6 +92,25 @@ public static void assertCollectable(WeakReference weakReference) {
         }
     }
 
+    public static void assertCollectable(WeakReference[] weakReferences) {
+        for (WeakReference wr : weakReferences) {
+            assertCollectable(wr);
+        }
+    }
+
+    public static <T> void assertCollectable(ArrayList<WeakReference<T>> weakReferences) {
```

Comment:
Could we change the argument to Collection<...>?

Reply:
Done